hypergan/configurations/improved-gan-resizable.json

```diff
@@ -30,8 +29,7 @@
   {
     "class": "function:hypergan.losses.standard_loss.StandardLoss",
     "reduce": "reduce_mean",
-    "improved": true,
-    "minibatch": true
+    "improved": true
```

Comment:
Minibatch doesn't work here.  Maybe we can output an error if `gan.batch_size()*2 != shape(net)[0]`. 

The reason is that we stack 3 values on the Discriminator instead of 2(X, reconstruction of X, and a random generated sample).